src/containers/home/home.component.jsx

```diff
@@ -74,6 +74,8 @@ const Home = (props) => {
             optionBName={post.option_b_name}
             hasUserVoted={hasUserVoted}
             totalVotes={post.total_votes}
+            postOwnerUsername={post.owner_username}
+            postOwnerAvatar={post.owner_img}
```

Comment:
It's important that the database be normalized. In other words, we have one source of truth for each piece of information. In this case the owner can just be stored as `owner: fa6sfdsf687sd6f8s7d6f` inside the `posts` entity in the database. So that if you need owner info you can fetch it from the `users/user-id` in the database as the single source of truth.

Reply:
This is where we need to get rid of these and just fetch this information based on uid?

Reply:
Yup when you're fetching the post, also need to follow up with a call to fetch the user